src/alert_transmitter.py

```diff
@@ -0,0 +1,25 @@
+class AlertTransmitter():
+
+    def __init__(self, alertTarget):
+        self.alertTarget = alertTarget
+        self.alerter_function = self.set_alerter_function()
+
+    def set_alerter_function(self,):
```

Comment:
naming of the method needs to reflect what it does, in this case returns alerter function. Hence it should be get... instead of set..()